src/explorer/CosmosDBTreeItem.ts

```diff
@@ -40,7 +40,7 @@ export class CosmosDBTreeItem extends AzureParentTreeItem<ISiteTreeRoot> {
         }
 
         if (ext.cosmosAPI === undefined) {
-            ext.cosmosAPI = <VSCodeCosmosDB>cosmosDB.exports;
+            ext.cosmosAPI = <CosmosDBExtensionApi>cosmosDB.exports;
```

Comment:
You're going to have to change this once you merge the cosmos db pr with master. It will return an [AzureExtensionApiProvider](https://github.com/Microsoft/vscode-azuretools/blob/master/ui/api.d.ts#L15) which you'll have to call `getApi` on and pass in `^1.0.0`